vali_objects/utils/position_manager.py

```diff
@@ -993,6 +993,15 @@ def save_miner_position_to_disk(self, position: Position, delete_open_position_i
 
         ValiBkpUtils.write_file(miner_dir + position.position_uuid, position)
 
+    def overwrite_position_on_disk(self, position: Position) -> None:
+        # delete the position from disk. Try the open position dir and the closed position dir
+        self.delete_position_from_disk(position, check_open_and_closed_dirs=True)
+        miner_dir = ValiBkpUtils.get_partitioned_miner_positions_dir(position.miner_hotkey,
+                                                                     position.trade_pair.trade_pair_id,
+                                                                     order_status=position.is_open_position,
```

Comment:
Should this be an OrderStatus enum?

Reply:
order_status=OrderStatus.OPEN if position.is_open_position else OrderStatus.CLOSED